Automattic/jetpack
# Fix for wp-missed-scheduled plugin

Fixes the compatibility with the wp-missed-scheduled plugin. 

The plugin published scheduled posts on certain hosts where cron is not working as expected.

To test:
Disable cron add to wp-config.php
define( 'DISABLE_WP_CRON', true );

Add the plugin https://github.com/sLaNGjI/wp-missed-schedule

Schedule a new post.
It should to email the subscribers. ( subscriptions should be on)

The subscribers should get an email.After the post is published.
